src/App.js

```diff
@@ -1,11 +1,23 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+//import React from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
+// *** React useState Hook *** //
+
+import React, {useState} from "react";
```

Comment:
Pretty straight forward project. You did step one, check. (Not a whole lot of ways to screw that up)